cmd/nex/workload.go

```diff
@@ -58,23 +60,23 @@ type NatsCreds struct {
 
 // Workload subcommands
 type RunWorkload struct {
-	NodeId      string            `description:"Node ID to run the workload on"`
-	NodeTags    map[string]string `description:"Node tags to run the workload on; --node-id will take precedence"`
-	NodePubXKey string            `name:"node-xkey-pub" description:"Node public xkey used for encryption"`
-
-	WorkloadName             string            `name:"name" description:"Name of the workload"`
-	WorkloadArguments        []string          `name:"argv" description:"Arguments to pass to the workload"`
-	WorkloadEnvironment      map[string]string `name:"env" description:"Environment variables to set for the workload"`
-	WorkloadDescription      string            `name:"description" description:"Description of the workload"`
-	WorkloadEssential        bool              `name:"essential" description:"Is the workload essential" default:"false"`
-	WorkloadHash             string            `name:"hash" description:"Hash of the workload. If provided, will use for validation, if not provided, will calculate"`
-	WorkloadHostServiceCreds NatsCreds         `embed:"" prefix:"hostservices." description:"Host service configuration"`
-	WorkloadJsDomain         string            `name:"jsdomain" description:"JS Domain to run the workload under"`
-	WorkloadRetryCount       int               `name:"retry-count" description:"Number of times to retry the workload" default:"3"`
-	WorkloadPublicKey        string            `name:"public-key" description:"Public key of the workload"`
-	WorkloadUri              string            `name:"uri" description:"URI of the workload.  file:// oci:// nats://" placeholder:"file://./workload"`
-	WorkloadTriggerSubjects  []string          `name:"triggers" description:"Subjects to trigger the workload"`
-	WorkloadType             string            `name:"type" description:"Type of workload" default:"direct_start"`
+	NodeId      string            `help:"Node ID to run the workload on"`
+	NodeTags    map[string]string `help:"Node tags to run the workload on; --node-id will take precedence"`
```

Comment:
Back to back caps are not split, so XKey results in `--xkey`
here's how the generation looks 
```
└─❯ go run . workloads run -h | grep xkey
      --node-xkey-pub=STRING       Node public xkey used for encryption
```

> For clarity I suggest adding an explicit name so there is no question on the semantics
I will create an issue for this, because that will be a lot of edits across a lot of files